cmake/IncludeGMock.cmake

```diff
@@ -128,9 +139,19 @@ elseif("${GOOGLE_CLOUD_CPP_GMOCK_PROVIDER}" STREQUAL "vcpkg")
     target_link_libraries(gmock INTERFACE GTest::gmock)
 
 elseif("${GOOGLE_CLOUD_CPP_GMOCK_PROVIDER}" STREQUAL "package")
-    find_package(Threads REQUIRED)
     find_package(GTest REQUIRED)
 
+    # FindGTest() is a standard CMake module. It, unfortunately, *only* creates
```

Comment:
It looks like these added blocks are identical -- I'm not terribly familiar with CMake's functions/macros, but could we factor these out into a `create_gtest_aliases` function and just call that function here and above? That would avoid potentially making changes in one spot but not the other in the future.